components/feature/addons/src/test/java/AddonCollectionProviderTest.kt

```diff
@@ -219,6 +219,17 @@ class AddonCollectionProviderTest {
         verify(cachingProvider).writeToDiskCache(jsonResponse)
     }
 
+    @Test
```

Comment:
Should we also add an extra test to verify that we are only deleting add-on collection files?

Reply:
Yeah, good idea. I've added a test for this now: https://github.com/mozilla-mobile/android-components/pull/8480/files#diff-c1cd858b6129959a62115bf8d2f4229cR235

Reply:
Looks great!